tools/uart_hw_test/uart_test_windows.c

```diff
@@ -35,9 +35,9 @@
  * Preprocessor definitions
  ****************************************************************************/
 #define UART_PATH_STR		"\\\\.\\COM%s"
-#define UART_PATH_STR_BELOW_TEN		"COM%s"
+#define UART_ALT_PATH_STR	"COM%s"
```

Comment:
If there is no way to have a common macro, please add a single line comment explaining which macro is for what.